doc/examples/segment_extending_clustering_framework.py

```diff
@@ -0,0 +1,233 @@
+"""
+===========================================================
+How to make new `Feature` and new `Metric` for Quickbundles
```

Comment:
I find the former better, for now. The second one might be goof once we will have more tutorials describing all the potential of the framework.